firmware/config/boards/48way/connectors/main.yaml

```diff
@@ -0,0 +1,106 @@
+pins:
+    - id: E2
+      class: outputs
+      ts_name: Fan Relay (E2)
+    - id: E3
+      class: outputs
+      ts_name: Fuel Relay (E3)    
+    - id: E4
+      class: analog_inputs
+      ts_name: Launch Control (E4)      
```

Comment:
launch control is not analog_inputs

Reply:
Having hard time referencing another board that is using LC

Reply:
```
  - pin: C4
    # H144_IN_VSS
    id: F11
    class: switch_inputs
    function: 2Step Digital Input
    type: hall
    ts_name: C4 - 2Step/Digital
    color: gray
```